lib/ui/map/map_action_buttons/route_button_bar.dart

```diff
@@ -42,7 +42,7 @@ class RouteButtonBar extends StatelessWidget {
       MapRouteState.LOADING => "Lade Route ...",
       MapRouteState.SHOWN ||
       MapRouteState.HIDDEN =>
-        "${(mapRoute.route!.distance / 1000).toStringAsFixed(2)} km | ${(mapRoute.route!.duration / 60).toStringAsFixed(0)} Min",
+        "${(mapRoute.route!.distance / 1000).toStringAsFixed(2)} km", // to long: | ${(mapRoute.route!.duration / 60).toStringAsFixed(0)} Min",
```

Comment:
@thomas-munichways ich find die Zeit ganz hilfreich, aber wir können das gerne vorerst so machen. Ich würde gerne als Nächstes generell die UI überarbeiten, damit man die Suche einfach findet und auch während der Fahrt noch mehr Infos einfach angezeigt bekommt. Aber das können wir dann ja nochmal anschauen, ich mal was auf. Bis dahin merge ich das hier, damit es so in den nächsten Release rausgeht.